gio/tests/dbus_peer.rs

```diff
@@ -0,0 +1,159 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(unix)]
+#[test]
+fn test_gdbus_peer_connection() {
+    use gio::{
+        glib::{self, VariantTy},
+        prelude::SocketExt,
```

Comment:
You have the prelude below, replace this with prelude::* and remove the other one

Reply:
Done